packages/lib/src/ui/blocks/PersonList/PersonList.tsx

```diff
@@ -0,0 +1,138 @@
+import { Block } from "../../components/Block/Block"
+import { Container } from "../../components/Container/Container"
+import styled from "@emotion/styled"
+import {
+  Header3,
+  SmallText,
+  Text,
+} from "../../components/Typography/Typography"
+import React from "react"
+import { PersonListProps } from "./personListDef"
+import { Image as ImageType } from "../../../types/data"
+
+import Circle1 from "../../assets/vectors/potatoes/circle_1.svg"
+import Circle2 from "../../assets/vectors/potatoes/circle_2.svg"
+import Circle3 from "../../assets/vectors/potatoes/circle_3.svg"
+import Circle4 from "../../assets/vectors/potatoes/circle_4.svg"
+import Circle5 from "../../assets/vectors/potatoes/circle_5.svg"
+import Circle6 from "../../assets/vectors/potatoes/circle_6.svg"
+import Circle7 from "../../assets/vectors/potatoes/circle_7.svg"
+import Circle8 from "../../assets/vectors/potatoes/circle_8.svg"
+
+import { Image } from "../../components/Image/Image"
+import { max, theme } from "../../styles/theme"
+
+const circles = [
+  Circle1,
+  Circle2,
+  Circle3,
+  Circle4,
+  Circle5,
+  Circle6,
+  Circle7,
+  Circle8,
+]
+
+export interface PersonProps {
+  name: string
+  nickname?: string
+  image: ImageType
+  subtitle?: string
+  contact?: {
+    email?: string
+    phone?: string
+  }
+}
+
+export const PersonList: React.FC<PersonListProps> = (props) => (
+  <Block>
+    <Container>
+      <PersonListContainer>
+        {props.content.map((person, index) => (
+          <Person key={index}>
+            <Picture
+              mask={circles[(index * 13 + 11) % 8].src}
```

Comment:
Co je tohle za zvláštní výpočet?

Reply:
Vybírám masky trochu náhodně (`* 13 + 11` mi dává pseudonáhodný pořadí, a `% 8` zajíšťuje že to nepřeteče délku `circles`. I když asi bude lepši využit přímo `length`, když to teď píšu.

Reply:
```ts
circles[Math.floor(Math.random() * circles.length)]
```

Reply:
A nebude takhle blok pokaždé jiné? To je důvod proč jsme random nepoužil.

Reply:
A jo. To máš pravdu, že není moc dobré. Mě se teda nelíbí ani takhle, že bude mít každá fotka jiný rámeček. Bude to zbytečně strhávat pozornost. Myslím, že by brambora měla být mnohem méně výrazná a všude stejná, nebo jen s drobnými nuancemi.

Reply:
Zkusím to upravit, tak se můžeš podívat. Fotky by už měli fungovat.

Reply:
Nezapomněl jsi to pushnout?

Reply:
Už to je